File: src/question/day0708/ms_0708.cpp
#include <iostream>

using namespace std;

int main() {
	const char* capital[] = { "Seoul", "Washington", "Tokyo", "Beijing" };
	const char* state[] = { "Korea", "USA", "Japen", "China" };

	while (true) {
		for (int i = 0; i < 4; i++) {
			cout << (i + 1) << ". " << *(state + i) << endl;
		}
		int cnt;

		cout << "number? ";
		cin >> cnt;
		
		switch (cnt) {
		case 1:
		case 2:
		case 3:
		case 4:
			cout << endl << *(capital + cnt - 1) << endl << endl;
			break;
		default:
			cout << endl << "none" << endl;
			return 0;
		}
	}
}

File: src/question/day0708/ms_0708.cs
using System;

namespace algorithm_08
{
    class ms_0708
    {
        public static readonly string[] state = { "Korea", "USA", "Japen", "China" };
        public static readonly string[] capital = { "Seoul", "Washington", "Tokyo", "Beijing" };

        static void Main(string[] args)
        {
            
            while(true) {
                for(var i = 0; i < 4; i++)
                    Console.WriteLine($"{i+1}. {state[i]}");

                Console.Write("number? ");
                int cnt = Convert.ToInt32(Console.ReadLine());

                if(1 <= cnt && cnt <= 4) 
                    Console.WriteLine($"\n{capital[cnt-1]}\n");
                else {
                    Console.WriteLine("\nnone\n");
                    break;
                }
            }
        }
    }
}
